src/Microsoft.AspNetCore.Razor.Tools/Application.cs

```diff
@@ -37,6 +39,8 @@ public Application(CancellationToken cancellationToken, ExtensionAssemblyLoader
 
         public ExtensionDependencyChecker Checker { get; }
 
+        public Func<string, MetadataReferenceProperties, PortableExecutableReference> AssemblyReferenceProvider { get; set; }
```

Comment:
Does this need to be get/set and passed in the constructor?

Reply:
Will remove the set.